src/run_environment/machine.ml

```diff
@@ -164,22 +164,22 @@ end
 type t = {
   name: string;
   host: Host.t;
-  reference_genome_dir: string;
+  pyensembl_cache_dir: string option;
   get_reference_genome: string -> Reference_genome.t;
   toolkit: Tool.Kit.t;
   run_program: Make_fun.t;
   work_dir: string;
   max_processors: int;
 }
 let create
-    ~host ~reference_genome_dir ~get_reference_genome ~toolkit
+    ~host ~pyensembl_cache_dir ~get_reference_genome ~toolkit
```

Comment:
should be an optional: `?pyensembl_cache_dir`

If not, it would force everybody to use explicitly: `~pyensembl_cache_dir:None`